### ISSUE TITLE
OQRA NaturalQuestions pre-trained model accuracy

### ISSUE THREAD

**Issue by priyamtejaswin** [google-research/language#97]
Hi folks,

Thanks a lot for releasing the source code and the pre-trained models!
I was able to setup everything, and run the `test` evaluation code.

I downloaded the pre-trained NQ checkpoints from `gs://orqa-data/orqa_nq_model`
When I run 
```bash
export MODEL_DIR=./orqa_nq_model
python -m language.orqa.predict.orqa_eval \
  --dataset_path=./resplit/WebQuestions.resplit.test.jsonl \
  --model_dir=$MODEL_DIR
```
I only see an accuracy of 21%
`Accuracy: 0.2101 (427/2032)`

This is using the `best_default` checkpoint:
https://github.com/google-research/language/blob/2d08af4d8bd1e744fa7ca3c176801653c2bc2b3a/language/orqa/models/orqa_model.py#L508-L509

The performance reported in the paper is 33.3%

Has anyone else faced this problem?
Did I miss a step?

Thanks,
Priyam

**Comment by kentonl** [comment]
The `dataset_path` that you used is for WebQuestions, not Natural Questions. Please use `resplit/NaturalQuestions.resplit.test.jsonl` instead.
